plugins/datamonitorPlugin/include/datamonitorplugin/datamonitormodel.hpp

```diff
@@ -65,8 +80,10 @@ class SCOPY_DATAMONITORPLUGIN_EXPORT DataMonitorModel : public QObject
 	void checkMinMaxUpdate(double value);
 	void setDataStorageSize();
 
-	IReadStrategy *readStrategy = nullptr;
-	UnitOfMeasurement *unitOfMeasure = nullptr;
+	IReadStrategy *readStrategy;
+	UnitOfMeasurement *unitOfMeasure;
+
+	bool m_isDummyMonitor = false;
```

Comment:
dummy monitors represent the loaded data and are treated different in the selection menu since they can be deleted. 
A dummy read strategy can be created but what would be the purpose for it ? the main idea behind a dummy monitor is that it can't get new data and it can be removed